plugins/modules/net_tools/pritunl/pritunl_org.py

```diff
@@ -0,0 +1,257 @@
+        type: str
+        required: true
+        description:
+            - API Token of a Pritunl admin user (It needs to be enabled in
+              Administrators > USERNAME > Enable Token Authentication).
+
+    pritunl_api_secret:
+        type: str
+        required: true
+        description:
+            - API Secret found in Administrators > USERNAME > API Secret.
+
+    state:
+        type: str
+        required: false
+        default: list
+        choices: [ list, present, absent ]
+        description:
+            - If C(list) is used, organizations will be retrieved from Pritunl,
+              C(name) can be used to filter by name, by default all of them
+              will be listed. If C(present), the organization C(name) will be
+              added to Pritunl. If C(absent), the organization C(name) will be
+              deleted from Pritunl.
+
+    name:
+        type: str
+        required: false
+        default: None
+        description:
+            - Name of the Organization to manage in Pritunl.
+
+    validate_certs:
+        type: bool
+        required: false
+        default : true
+        description:
+            - If certificates should be validated or not.
+'''
+
+EXAMPLES = '''
+# List all existing organizations
+- name: List all existing organizations
+  pritunl_org:
+    state: list
+
+# Gather information about the organization named MyOrg
+- name: Gather information about the organization named MyOrg
+  pritunl_org:
+    state: list
+    name: MyOrg
+
+# Make sure the organization named Org1 exists
+- name: Make sure the organization named Org1 exists
+  pritunl_org:
+    state: present
+    name: Org1
+
+# Make sure the organization named Org1 does not exist
+- name: Make sure the organization named Org1 does not exist
+  pritunl_org:
+    state: absent
+    name: Org1
+'''
+
+RETURN = '''
+response:
+    description: JSON representation of Pritunl Organizations
+    returned: success
+    type: list
+    sample: [
+        {
+            "auth_api": false,
+            "auth_secret": null,
+            "auth_token": null,
+            "id": "58070daeec3a3b2e6e472c36",
+            "name": "Org1",
+            "user_count": 10
+        },
+        {
+            "auth_api": false,
+            "auth_secret": null,
+            "auth_token": null,
+            "id": "58070s38e63f3c2e6e4x2d5f",
+            "name": "Org2",
+            "user_count": 0
+        },
+    ]
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.basic import json
+from ansible.module_utils.six import iteritems
+from ansible_collections.community.general.plugins.module_utils.net_tools.pritunl.api import pritunl_auth_request
+
+
+def _list_pritunl_organization(module, filters=None):
+    orgs = []
+
+    response = pritunl_auth_request(module, 'GET', '/organization')
+
+    if response.getcode() != 200:
+        module.fail_json(msg='Could not retrive organizations from Pritunl')
```

Comment:
```suggestion
        module.fail_json(msg='Could not retrieve organizations from Pritunl')
```
We could put in places like this some useful information here, e.g. the ret code and/or error message (use `to_native()` from `module_utils` to wrap around the message))